internal/pkg/aws/cloudformation/changeset.go

```diff
@@ -0,0 +1,160 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cloudformation
```

Comment:
Also just thinking out loud. This seems very generally useful and might be nice to put in pkg/ so other folks can use it. We don't have to do it now, but I'm just thinking about moving forward what can we expose to customers. I mean, even just moving this CF stuff to it's own pkg might make sense, because as you mentioned, CF is surprisingly difficult to 'get right'.